monorepo/monorepo-ws/libs/prisma-article-app-client/prisma/schema.prisma

```diff
@@ -2,8 +2,9 @@
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
 generator client {
-  provider = "prisma-client-js"
-  output = "../src/generated"
+  provider      = "prisma-client-js"
+  output        = "../src/generated"
+  binaryTargets = ["native", "linux-musl"]
```

Comment:
hele, tohle nakonec asi bude zbytečný. protože sme to zkoušeli jen ze zoufalství když prisma nešla :-D Jelikož se to bude generovat v dockeru, tak by to mohlo jít bez týhle změny.

Reply:
bez toho to nejede